### PR TITLE
Fix retry loop

### PR DESCRIPTION
**Pull request by dsellarsnr**
* Handle case where refresh token invalid - logout user and return to login screen

### CODE REVIEW

**Review by marcelofarias — APPROVED** [review]

File: jb/src/main/kotlin/agent/CodeStreamLanguageClient.kt [hunk 1]
```diff
@@ -136,6 +136,13 @@ class CodeStreamLanguageClient(private val project: Project) : LanguageClient {
     @JsonNotification("codestream/didFailLogin")
     fun didFailLogin(json: JsonElement?) {}
 
+    @JsonNotification("codestream/didEncounterInvalidRefreshToken")
```
Comment (bcanzanella) [inline jb/src/main/kotlin/agent/CodeStreamLanguageClient.kt:139]: do we need also need this for VS?

Comment (dsellarsnr) [reply jb/src/main/kotlin/agent/CodeStreamLanguageClient.kt:139]: yes, meant to ask @CalvinAllen

Comment (dsellarsnr) [reply jb/src/main/kotlin/agent/CodeStreamLanguageClient.kt:139]: should we merge this and to VS in a separate PR?

Comment (bcanzanella) [reply jb/src/main/kotlin/agent/CodeStreamLanguageClient.kt:139]: @dsellarsnr yes

**Pull request merged**